services/ai-navigator-app/0.2.4/helmrelease/helmrelease.yaml

```diff
@@ -23,4 +23,4 @@ spec:
       retries: 30
   valuesFrom:
     - kind: ConfigMap
```

Comment:
`version: 0.2.1` needs to be bumped, and a new chart needs to be released for the `values.yaml` update. It's manual right now. I'll release `version: 0.2.2` Please bump to that.

Reply:
> `version: 0.2.1` needs to be bumped, and a new chart needs to be released for the `values.yaml` update. It's manual right now. I'll release `version: 0.2.2` Please bump to that.

Bumped.

Reply:
Another inconsistency that was preventing me from updating the license file:

This PR has `ghcr.io/mesosphere/dkp-container-images/cr.weaviate.io/semitechnologies/weaviate:1.21.4`
the [AI Nav API now has](https://github.com/mesosphere/ai-navigator-cluster-info-api/blob/5b16b4698e304b4e15c635684146a5d26e2d264b/chart/cluster-info-api/values.yaml#L29-L31) `ghcr.io/mesosphere/dkp-container-images/cr.weaviate.io/semitechnologies/weaviate:1.21.4-d2iq.0`

Which one is it? These need to be consistent, and the license file needs to be updated.

Same problem with `ghcr.io/mesosphere/dkp-container-images/docker.io/bitnami/postgresql:15.2.0-debian-11-r21` and `ghcr.io/mesosphere/dkp-container-images/docker.io/bitnami/postgresql:15.2.0-debian-11-r21-d2iq.0`

Reply:
Great catch, there was the `tag` value in the defaults CM which was overriding helm default.

```
    # Default values for subcharts
    weaviate:
      enabled: true
      globalPriorityClassName: dkp-high-priority
      service:
        type: ClusterIP
      image:
        tag: 1.21.4
```